js/modules/input-mask.js

```diff
@@ -302,7 +302,8 @@ const inputMask = new function () {
     }
 
     const getMaskedValue = (inputEl) => {
-        const { dataset, value } = inputEl;
+        const { dataset } = inputEl;
+        const value = inputEl.value || '';
```

Comment:
Is there a reason that the following won't work? That's the normal way we set defaults when destructuring objects.

```suggestion
        const { dataset, input = '' } = inputEl;
```